.github/workflows/ci.yml

```diff
@@ -122,7 +122,26 @@ jobs:
           run: |
             export TEST_NAMESPACE=shipwright-build
             export TEST_IMAGE_REPO=registry.registry.svc.cluster.local:32222/shipwright-io/build-e2e
+            if [ "${{ secrets.SAMPLE_RUBY_PRIVATE_SSHKEY }}" != "" ]; then
```

Comment:
The name of the secret is a little confusing as it talks about Ruby while the sample project is NodeJS. So, if you get hold of somebody who can create a secret with a suitable name instead, that would be nice. Then the secret can also be without base64 encoding as we tried it out earlier today.

Reply:
Yes, I am requesting @adambkaplan to help rename it, will update the code accordingly after it's changed.